prototypes/recipes/cadaveric-arum/recipes-auto-cadaveric.lua

```diff
@@ -48,7 +48,7 @@ FUN.autorecipes {
 				{
                     {name='cadaveric-arum-seeds', add_amount = 10},
                     {name='pesticide-mk01', amount = 2},
-					{name='acidgas', amount = 50},
+					{name='acidgas', amount = 50, type = "fluid", fluidbox_index = 3},
```

Comment:
You add type=fluid here but not elsewhere, why?

Reply:
I didn't understand how the auto recipes worked at first. Removed